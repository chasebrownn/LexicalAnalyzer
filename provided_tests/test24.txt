t1 ((((a).(b)).((a)*)).((b)*))* ,  t2 (((a)|(b))*).(((c)|(d))*) #
" abaaabb "