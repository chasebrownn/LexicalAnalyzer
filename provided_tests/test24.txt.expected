EPSILON IS NOOOOOT A TOKEN !!! t1 t2